custom/icds_reports/views.py

```diff
@@ -1649,3 +1651,12 @@ def post(self, request, *args, **kwargs):
                     messages.success(request, _('Successfully enqueued request to submit files for translations'))
                     return redirect(self.urlname, domain=self.domain)
         return self.get(request, *args, **kwargs)
+
+
+@method_decorator([login_and_domain_required], name='dispatch')
+class InactiveAWW(View):
+    def get(self, request, *args, **kwargs):
+        collect_inactive_awws()
```

Comment:
Why does this need to be called? I think that that would kick off the aggregation and be very expensive